test-harness/local-run.sh

```diff
@@ -1,9 +1,11 @@
-#!/bin/bash
+#!/usr/bin/env bash
 #---------- see https://github.com/joelong01/BashWizard ----------------
 # bashWizard version 1.0.0
 # this will make the error text stand out in red - if you are looking at these errors/warnings in the log file
 # you can use cat <logFile> to see the text in color.
 
+export FLAGS_GETOPT_CMD="$(brew --prefix gnu-getopt)/bin/getopt"
```

Comment:
```bash
$ export FLAGS_GETOPT_CMD="$(brew --prefix gnu-getopt)/bin/getopt"

Command 'brew' not found, but can be installed with:

sudo apt install linuxbrew-wrapper
```

Can we wrap this in an OS check?
What happens if the user does not have brew?

Reply:
If the user does not have brew, the brew install for gnu-getopt would fail as well. Was going to bring this up to Joe to see if we should include an OS or dependency check in BashWizard, then propagate that here. As of right now, this PR is a WIP until a direction for the Wizard is settled on.